### PR TITLE
Added Pro Serve Job Description

### PR DESCRIPTION
**Pull request by pete0emerson**

### CODE REVIEW

File: pages/careers/_open-positions.html [hunk 1]
```diff
@@ -96,6 +97,76 @@ <h4>Your Ideal Background</h4>
...
+      </li>
+      <li>Bonus points for a sense of humor, empathy, and curiosity.</li>
+      <li>
+        Note that we're less concerned with prior experience than we are with
+        curiosity about all areas of the stack and demonstrated ability to learn
+        quickly and go deep when necessary.
+      </li>
+    </ul>
+
+    You'll write plenty of code:
```
Comment (brikis98) [inline pages/careers/_open-positions.html:162]: Copy/paste error?

**Review by brikis98 — COMMENTED** [review]
LGTM, just one stray copy/paste thing to fix!

File: pages/careers/_open-positions.html (hunk 1 above)
Comment (pete0emerson) [reply pages/careers/_open-positions.html]: Fixed, and added URL for Workable.

**Review by brikis98 — DISMISSED** [review]
Ship!

**Comment by pete0emerson** [comment]
So sorry, my Apply Now URL was malformed. Fixed now.

**Review by brikis98 — APPROVED** [review]
LGTM

**Pull request merged**